README.md

```diff
@@ -10,6 +10,46 @@ intended for testing new features, and can deploy sandboxes automatically from
 GitHub pull requests.
 
 
+Using the Ocim web console
```

Comment:
I have just one last comment about this: should this whole section be moved down _after_ the install part? Whenever I look at a repo for some software, I care about 2 things right away: what is it, and how do I install it?

The "what is it" part is answered above this section, and the "how do I install it" part is below. This part seems to get in the way and throws too many intricacies about Ocim to the reader before they even know how to get started.

Reply:
I want the document clearly divided in two parts: the user part and the developer part. It could be a different page if you want.
I put the user part first because everyone needs it in the beginning, and because too often I have been lost in OpenCraft's documentation because it has too many sections.
If a random developer browsing GitHub doesn't know Ocim yet, he still has to know what it does and if he likes it or not, before installing it. This „user“ part should be the one that sells it. We could say that the screenshot is enough for that and we could move the screenshot to before this header so that the usage section is smaller.
Or we can add some short links at the top to directly go to the „Install“ header (or to all key sections: install, settings, run, debug, tests).

Reply:
@clemente Yeah I get your point.

I had the same alternative idea: the image is a good "seller" here, so we can just move that up. I'd be good with that approach. A table of contents would also be really helpful, if you have time for it. We can delay implementing a ToC part until later in the week if you don't have time right now, it's not as urgent.

Reply:
Ok, I moved the image and added a very simple TOC.